app/src/main/java/org/simple/clinic/patient/recent/RecentPatient.kt

```diff
@@ -55,12 +55,14 @@ data class RecentPatient(
           LEFT JOIN (
             SELECT MAX(createdAt) latestCreatedAt, T.*
```

Comment:
Can you specify the fields you need instead of using `T.*`?

Reply:
Let's not choose to do something because it is concise or easy.
We have similar query in `Patient#RoomDao` in `mainQuery` and in other places like `PatientSearchResult`.
Please refer those for making this change.

Reply:
done